packaging/python/test_wheels.sh

```diff
@@ -301,10 +301,7 @@ $python_exe -m pip install --upgrade pip
 # install numpy, pytest and neuron
 $python_exe -m pip install numpy pytest
 $python_exe -m pip install $python_wheel
-$python_exe -m pip show neuron \
-    || $python_exe -m pip show neuron-nightly \
-    || $python_exe -m pip show neuron-gpu \
-    || $python_exe -m pip show neuron-gpu-nightly
+$python_exe -m pip show neuron || $python_exe -m pip show neuron-nightly
 
```

Comment:
Could you also remove relevant bits from `packaging/python/README`?

Reply:
Done